src/calculateRentalCost.js

```diff
@@ -29,6 +29,15 @@
 
 function calculateRentalCost(days) {
   // write code here
+  const price = 40;
+
+  if (days >= 3 && days < 7) {
+    return price * days - 20;
+  } else if (days >= 7) {
+    return price * days - 50;
+  }
```

Comment:
Use variables for the main values so that you can reuse them and give them descriptive names